Formula/apache-ctakes.rb

```diff
@@ -0,0 +1,85 @@
+class ApacheCtakes < Formula
+  desc "NLP system for extraction of information from EMR clinical text."
+  homepage "https://ctakes.apache.org"
+  url "https://apache.osuosl.org/ctakes/ctakes-4.0.0/apache-ctakes-4.0.0-bin.tar.gz"
+  sha256 "37ca2b8dfe06465469ed1830fbb84dfc7bcc4295e5387d66e90a76ad2a5cdeaf"
+
+  bottle :unneeded
+
+  option "with-umls", "Install UMLS Dictionary"
```

Comment:
No. You have to download it separately – see https://ctakes.apache.org/downloads.cgi .